python/social_bot/envs/grocery_ground.py

```diff
@@ -70,7 +70,7 @@ class GroceryGroundGoalTask(GroceryGroundTaskBase, GoalTask):
 
     def __init__(self,
```

Comment:
The default "__init__()" parameter could be modified to be the same as initializing code, to keep the default behavior the same after removed the initialing code.

Reply:
Done.